internal/server/publication.go

```diff
@@ -1138,3 +1140,258 @@ func (s *server) CleanupPublications(req *api.CleanupPublicationsRequest, stream
 
 	return nil
 }
+
+func sendSyncPublicationContributorsError(stream api.Biblio_SyncPublicationContributorsServer, e error) error {
```

Comment:
Move this into `SyncPublicaitonContributors`.  The gRPC error handling is too specific to turn into separate functions that are maybe called 2 or 3 times from the same function. Avoiding duplicate code here for the sake of readability is just not worth it.

Reply:
it is called 8 times ;-)